src/cascadia/TerminalSettingsModel/IInheritable.h

```diff
@@ -79,16 +79,9 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
     using NullableSetting = std::optional<std::optional<T>>;
 }
 
-// Use this macro to quickly implement both getters and the setter for an
-// inheritable setting property. This is similar to the WINRT_PROPERTY macro, except...
-// - Has(): checks if the user explicitly set a value for this setting
-// - SourceGetter(): return the object that provides the resolved value
-// - Getter(): return the resolved value
-// - Setter(): set the value directly
-// - Clear(): clear the user set value
-// - the setting is saved as an optional, where nullopt means
-//   that we must inherit the value from our parent
-#define INHERITABLE_SETTING(projectedType, type, name, ...)                 \
+// Shared implementation between the INHERITABLE_SETTING and INHERITABLE_NULLABLE_SETTING macros.
+// See below for more details.
+#define _BASE_INHERITABLE_SETTING(projectedType, storageType, name, ...)    \

```

Comment:
That's a reserved identifier. But it's fine...

Reply:
Your mom's a reserved identifier.